internal/clients/imagebuilder/image_builder_client.go

```diff
@@ -0,0 +1,80 @@
+package imagebuilder
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+)
+
+const URL = "https://console.redhat.com/api/image-builder/v1/compose"
```

Comment:
The idea is to use IB directly, so I assume we need to make the URL configurable and then OpenShift will fill the right URL in for us? Or is that a constant for all environments?

Reply:
It's an openshift feature, for openshift services it's `<service>.<pod_namespace>.svc.cluster.local`; so that would be `image-builder.image-builder-(prod|stage).svc.cluster.local` (initial url I gave isn't correct).

The url can be mapped using env variables, via openshift template parameters per namespace in app-intferface (they're predefined there).